CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Update TypeScript typings to Cloud Native [#147](https://github.com/CartoDB/carto-react/pull/147)
 - Remove SourceTypes from react-api public api [#147](https://github.com/CartoDB/carto-react/pull/147)
+- Add support to Cloud Native SQL API [#150](https://github.com/CartoDB/carto-react/pull/150)
 
 ## (prerelease) 1.1.0-alpha.2 (2021-05-28)
 ## (prerelease) 1.1.0-alpha.1 (2021-05-27)
```

Comment:
We might probably want to remove the alphas from changelog

Reply:
Yeah